code/modules/antagonists/nukeop/nukeop.dm

```diff
@@ -35,9 +41,15 @@
 	if(!nukeop_outfit) // this variable is null in instances where an antagonist datum is granted via enslaving the mind (/datum/mind/proc/enslave_mind_to_creator), like in golems.
 		return
 
-	operative.set_species(/datum/species/human) //Plasmamen burn up otherwise, and besides, all other species are vulnerable to asimov AIs. Let's standardize all operatives being human.
+	// If our nuke_ops_species pref is set to TRUE, (or we have no client) make us a human
+	if(!operative.client || operative.client.prefs.read_preference(/datum/preference/toggle/nuke_ops_species))
+		operative.set_species(/datum/species/human)
+
+	if(is_species(operative, /datum/species/plasmaman))
+		operative.equipOutfit(plasmaman_outfit)
```

Comment:
This should not be a hardcoded thing, it should defer to the species, IE via `pre_equip_species_outfit`. 
If we (or downstreams) wish to add a new species which requires its own unique outfits, we shouldn't have to come back to this to add `isplasmaman, elsif snowflakeman, elseif fishman, else...`

(this is the reason why I used jobs datums, it worked with that proc nicely)